twilio-authenticator-sample/src/main/AndroidManifest.xml

```diff
@@ -20,21 +20,35 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
         <activity
-            android:name=".main.MainActivity"
-            android:label="@string/menu_navigation_requests"
-            android:screenOrientation="portrait"
-            android:theme="@style/AppTheme.NoActionBar" />
+            android:name=".appslist.AppsActivity"
+            android:label="@string/title_activity_apps_list"
+            android:theme="@style/AppTheme.NoActionBar"
+            android:screenOrientation="portrait">
+        </activity>
+
         <activity
             android:name=".approvalrequests.detail.ApprovalRequestDetailActivity"
             android:label="@string/title_activity_approval_request_detail"
-            android:parentActivityName=".main.MainActivity"
+            android:parentActivityName=".appslist.MainActivity"
             android:screenOrientation="portrait">
             <meta-data
                 android:name="android.support.PARENT_ACTIVITY"
-                android:value=".main.MainActivity" />
+                android:value=".appslist.AppsActivity" />
         </activity>
 
+        <activity
+            android:name=".appslist.MainActivity"
```

Comment:
I don't like `appslist` as a package name, feels weird could you rename to just `apps`?